### PR TITLE
Updated models of Health Care Management

### PR DESCRIPTION
**Pull request by krishnanjali25**

### CODE REVIEW

File: FusionIIIT/applications/academic_information/models.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
+from applications.globals.models import ExtraInfo
 from django.db import models
 
-from applications.globals.models import ExtraInfo
 
 # Create your models here.
```
Comment (arshadsid) [inline FusionIIIT/applications/academic_information/models.py:6]: You do not have to make this unnecessary change

File: FusionIIIT/applications/health_center/models.py [hunk 2]
```diff
@@ -1,3 +1,112 @@
...
+            ('19', '7 p.m.'),
+            ('20', '8 p.m.'),
+            ('21', '9 p.m.'),
+            ('22', '10 p.m.'),
+            ('23', '11 p.m.'),
+            )
+
+
+class Doctor:
+    specialization = models.CharField(max_length=10, choices=Constants.SPECIALIZATION)
```
Comment (arshadsid) [inline FusionIIIT/applications/health_center/models.py]: Your models are incomplete

**Comment by krishnanjali25** [comment]
Flake and isort error sorted

**Review by arshadsid — CHANGES_REQUESTED** [review]
Doctor table is incomplete. Make necessary changes and commit again.

File: FusionIIIT/applications/academic_information/models.py (hunk 1 above)
Comment (krishnanjali25) [reply FusionIIIT/applications/academic_information/models.py:6]: you said that type is not needed

Comment (arshadsid) [reply FusionIIIT/applications/academic_information/models.py:6]: Yes but there were other fields as well

Comment (krishnanjali25) [reply FusionIIIT/applications/academic_information/models.py:6]: no there is just it , specialization and type

Comment (arshadsid) [reply FusionIIIT/applications/academic_information/models.py:6]: Add:
doctor_name
doctor_phone

**Comment by arshadsid** [comment]
Ok then no problem.
Models are correct.

**Comment by arshadsid** [comment]
@krishnanjali25 make your PR flake8 and isort compliant.

File: FusionIIIT/applications/health_center/models.py [hunk 3]
```diff
@@ -1,3 +1,114 @@
...
+            ('18', '6 p.m.'),
+            ('19', '7 p.m.'),
+            ('20', '8 p.m.'),
+            ('21', '9 p.m.'),
+            ('22', '10 p.m.'),
+            ('23', '11 p.m.'),
+            )
+
+
+class Doctor:
```
Comment (Samy-33) [inline FusionIIIT/applications/health_center/models.py:48]: This class doesn’t subclasses `models.Model`

File: FusionIIIT/applications/health_center/models.py [hunk 4]
```diff
@@ -1,3 +1,114 @@
...
+    medicine_id = models.ForeignKey(Stock)
+
+
+class Appointment(models.Model):
+    user_id = models.ForeignKey(ExtraInfo)
+    doctor_id = models.ForeignKey(Doctor)
+    description = models.CharField(max_length=50)
+    approval = models.BooleanField()
+    appointment_date = models.DateField()
+    appointment_time = models.CharField(max_length=10, choices=Constants.TIME)
```
Comment (Samy-33) [inline FusionIIIT/applications/health_center/models.py:96]: These two fields can be merged into one `datetime` field

**Comment by Samy-33** [comment]
Left some comments, and also look for flake8 and isort errors

**Comment by krishnanjali25** [comment]
screenshot 2

The flake error is because of models in globals

**Comment by arshadsid** [comment]
ran the checks on the current version of the master and no error in the master was found.
The changes from your PR are causing it.
Fix it in your repo before pushing.

**Comment by arshadsid** [comment]
also the main master in 3peers does not accept a PR if it does not clear the errors.

File: FusionIIIT/applications/globals/models.py [hunk 5]
```diff
@@ -2,6 +2,7 @@
 from django.contrib.auth.models import User
 from django.db import models
 
+
 # Class definations:
```
Comment (arshadsid) [inline FusionIIIT/applications/globals/models.py:6]: the error appears below this line. Changed by your commit.

**Comment by arshadsid** [comment]
not correct yet....
